src/ResourceManager/Sql/Commands.Sql/Failover Group/Services/AzureSqlFailoverGroupAdapter.cs

```diff
@@ -111,19 +134,14 @@ internal AzureSqlFailoverGroupModel UpsertFailoverGroup(AzureSqlFailoverGroupMod
             ReadWriteEndpoint readWriteEndpoint = new ReadWriteEndpoint();
             readWriteEndpoint.FailoverPolicy = model.ReadWriteFailoverPolicy;
 
-            if (model.FailoverWithDataLossGracePeriodHours.HasValue && !string.Equals(model.ReadWriteFailoverPolicy, FailoverPolicy.Manual.ToString()))
-            {
-                readWriteEndpoint.FailoverWithDataLossGracePeriodMinutes = model.FailoverWithDataLossGracePeriodHours * 60;
-            }
-            else
+            if (model.FailoverWithDataLossGracePeriodHours.HasValue)
             {
-                readWriteEndpoint.FailoverWithDataLossGracePeriodMinutes = null;
+                readWriteEndpoint.FailoverWithDataLossGracePeriodMinutes = checked(model.FailoverWithDataLossGracePeriodHours * 60);
```

Comment:
'Hours' is representative of what we actually support. We think making it 'Hours' in PowerShell regardless of the API is the best option to avoid misleading users. We intend to change the API.